Makefile

```diff
@@ -2,6 +2,9 @@
 
 all: quality test
 
+clean:
```

Comment:
Nope, we don't have to clean after the build and moreover `node_modules` should be cached